src/controllers/MasterController.php

```diff
@@ -1,6 +1,6 @@
 <?php
```

Comment:
Since the MasterController is a framework component, rather than an app controller, I would typically give it a unique namespace and location (e.g. src/FrontController/MasterController)